llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1473,6 +1477,11 @@ class LoopVectorizationCostModel {
   /// vscale_range.min == vscale_range.max then return vscale_range.max, else
   /// return the value returned by the corresponding TTI method.
   void initializeVScaleForTuning() {
+    if (VScaleForTuningOpt.getNumOccurrences()) {
+      VScaleForTuning = VScaleForTuningOpt;
+      return;
+    }
```

Comment:
Is the purpose of the option to deliberately override the function attribute?


Does this have to be LV specific? Could this be handled in `getVScaleForTuning`?